mr/src/main/java/org/opensearch/hadoop/rest/RestClient.java

```diff
@@ -776,33 +738,21 @@ public ClusterInfo mainInfo() {
         }
         String versionNumber = versionBody.get("number");
         OpenSearchMajorVersion major = OpenSearchMajorVersion.parse(versionNumber);
-        if (major.before(OpenSearchMajorVersion.V_6_X)) {
+        if (false && major.before(OpenSearchMajorVersion.V_6_X)) {
```

Comment:
That's interesting, `OpenSearchMajorVersion.V_6_X` should be probably rebranded as `ElastisearchMajorVersion.V_6_X` ?

Reply:
Hadoop Connector will be released w/ OpenSearch 3.0+ so it won't support Elasticsearch at all. There are a LOT of `ES_` variables that need to be refactored so this whole effort is being done in multiple PRs and phases rather than one monolithic PR.